hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/AbstractRootedOzoneFileSystemTest.java

```diff
@@ -325,6 +326,19 @@ void testCreateDoesNotAddParentDirKeys() throws Exception {
     fs.delete(grandparent, true);
   }
 
+  @Test
+  public void testCreateKeyWithECReplicationConfig() throws Exception {
+    String testKeyName = "testKey";
+    Path testKeyPath = new Path(bucketPath, testKeyName);
+    createKeyWithECReplicationConfiguration(cluster.getConf(), testKeyPath);
+
+    OzoneKeyDetails key = getKey(testKeyPath, false);
+    assertEquals(HddsProtos.ReplicationType.EC,
+        key.getReplicationConfig().getReplicationType());
+    assertEquals("rs-3-2-1024k",
+        key.getReplicationConfig().getReplication());
```

Comment:
These assertions are still duplicated between the test classes.  Please move these to:

https://github.com/apache/ozone/blob/4a4a7d5a4a43af1633460449538779797abe67ac/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/OzoneFileSystemTests.java#L102

Reply:
Thanks for updating the patch.  I just `getKey()` does not use `FileSystem` API, and the implementation is different between OFS and O3FS.  So I updated the patch to keep the assertions in the specific test classes.